manifests/install/all-in-one.yaml

```diff
@@ -0,0 +1,83 @@
+# First part
+# Apply extra privileges to system:kube-scheduler.
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:kube-scheduler:plugins
+rules:
+- apiGroups: ["scheduling.sigs.k8s.io"]
+  resources: ["podgroups", "elasticquotas"]
+  verbs: ["get", "list", "watch", "create", "delete", "patch"]
```

Comment:
I missed `patch` here; otherwise scheduler reports error:

```
E0317 01:20:22.700494       1 core.go:203] podgroups.scheduling.sigs.k8s.io "pg1" is forbidden: User "system:kube-scheduler" cannot patch resource "podgroups" in API group "schedul
ing.sigs.k8s.io" in the namespace "default"
```

Reply:
yep, we miss patch here